detect_secrets/pre_commit_hook.py

```diff
@@ -65,8 +65,8 @@ def main(argv=None):
 
     if baseline_modified:
         write_baseline_to_file(
-            args.baseline[0],
-            baseline_collection.format_for_baseline_output(),
+            filename=args.baseline[0],
+            data=baseline_collection.format_for_baseline_output(),
```

Comment:
🚨 Note that here we have to call `format_for_baseline_output` even though it'll be called again in `write_baseline_to_file ` because there is no other existing good way to create a dict from a `SecretsCollection` object, b/c `.json` only worries about `.data` (`['results']`) and not other attributes. Creating a new one didn't seem worth it.